nix/mobile/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, pkgs, target-os ? "", status-go, androidPkgs }:
+
+with pkgs;
+with stdenv;
+
+let
+  gradle = gradle_4_10;
```

Comment:
I don't get where this comes from, this is a string or a variable?

Reply:
`gradle_4_10` is a variable that comes from nixpkgs.

Reply:
I see, thanks!